docs/01-engage/02-accounts/02-creating-accounts/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Creating Accounts
+sidebar_position: 2
+---
+
+Accounts in **ENGAGE** are separate individuals (donors or supporters) and organizations that donate charity. This article will walk you through creating a new account via ENGAGE.
+
+## Option 1:1 Create an Account from the Dashboard using Create
+
+- On the [ENGAGE dashboard](https://beta.n3o.cloud/eu1/engage/dashboard), click the blue button **Create**. A bar containing different icons will be shown.
+
+![Create Account Dashboard](./create-account-dashboard.png)
+
+- Click the **Create Account** button. This will take you to the create account screen.
+
+## Option 1:2 Create an Account from the Dashboard using Quick Navigation
+
+- On the [ENGAGE dashboard](https://beta.n3o.cloud/eu1/engage/dashboard), navigate to **Quick Navigation**.
+
+![Create Account Quick Navigation](./create-account-quick-navigation.png)
+
+- Click on **Create Account** and a screen will appear to create the account.
+
+:::note 
+An account can be created for both individual and an organization. Each of them will have different fields to be covered. To follow along, make sure to have the correct information. 
+:::
+
+## Create an Individual Account
+
+:::note Important 
+- Input boxes with *asterisk* are compulsory to fill in. If you do not, an error message will appear e.g. *"State is a required field"*. 
```

Comment:
This isn't accurate if we include any screenshots. What is mandatory is different on different subscriptions.

I also would make the example of validation clearer lower down the page when talking about pressing the create button. At the point mention you may see one or more validation messages in Engage indicating missing or incorrect data that must be specified or corrected to proceed.